site/en/blog/new-in-chrome-109/index.md

```diff
@@ -71,7 +71,7 @@ This covers only some key highlights. Check the links below for
 additional changes in Chrome 109.
 
 * [What's new in Chrome DevTools (109)](/blog/new-in-devtools-109/)
-* [Chrome 109 deprecations and removals](/blog/deps-rems-109/)
+* [Chrome 109 deprecations and removals](/blog/chrome-109-beta/#deprecations-and-removals)
```

Comment:
is this for the beta version? it looks like it should be for the non-beta one. thanks for investigating :) 


<img width="1233" alt="Screenshot 2023-01-24 at 4 09 20 PM" src="https://user-images.githubusercontent.com/5950956/214441403-2d139f03-fc63-4c19-a383-9861fc1be099.png">